templates/includes/records/hierarchy-local.html

```diff
@@ -8,7 +8,13 @@
                     <p class="hierarchy-local__nav-item">
                         <span class="hierarchy-local__nav-indicator hierarchy-local__nav-indicator--left"></span>
                         <span class="hierarchy-local__nav-label">{{ 'previous_record'|as_label }}</span>
-                        <a href="{% record_url page.previous_record %}" data-link="Previous" class="link--bold"><span class="sr-only">{{ 'previous_record'|as_label }}</span> {{ page.previous_record.iaid }}</a>
+                        <a href="{% record_url page.previous_record %}" data-link="Previous" class="link--bold">
+                            {% if page.previous_record.has_reference_number %}
+                                <span class="sr-only">{{ 'previous_record'|as_label }}</span> {{ page.previous_record.reference_number }}
+                            {% else %}
+                                Previous {{ page.level.lower|default:"record" }}
```

Comment:
It's important that the 'Previous' text is part of the link, as that will be read out to screen readers

Reply:
> It's important that the 'Previous' text is part of the link, as that will be read out to screen readers

1. Why do we not use `{{ 'previous_record'|as_label }}` instead of Hard coded label?
2. Do you want to also add it to existing code where reference number is used?
3. `page.level.lower` could we just retain the casing returned in the response.

Same comments apply for `next record' as they are both similar

Reply:
1. Have updated to use `{{ 'previous_record'|as_label }}` and `{{ 'next_record'|as_label }}`
2. No, the better approach is to hide the 'hidden on mobile' label, and always have "Next thing" or "Previous thing" as the visible link text
3. GDS tend to prefer title casing, and since these are standard archival terms, and not people or place names, they shouldn't be capitalised. Waiting for confirmation on Slack thread.